_data/changelogs/about-policies.yml

```diff
@@ -2,6 +2,20 @@ title: Website policies
 type: documentation
 changelogURL:
 items:
+  - date: NNNN-NN-NN
```

Comment:
I don't think we need changelogs for these changes since it is more of a formatting issue rather than a "meaningful" change for the user. My vote is to leave them off, but curious if anyone objects!

Reply:
I'm also open to this! I don't think it adds much value to the history of the page.

Reply:
@mejiaj Does this sound alright with you? I'll go ahead and remove if so

Reply:
Yeah, that makes sense. Especially with the effort to create a changelog.

Reply:
Removed in [7161322](https://github.com/uswds/uswds-site/pull/2659/commits/71613228132612b21885c024ec3a573d750b0f11)